tests/ops/util/test_connection_type.py

```diff
@@ -0,0 +1 @@
+import pytest
```

Comment:
this looks like a mistake, since it's just this one line?

that being said, not sure if we do need to update any testing to accommodate your change to include the new `connection_type.py` util file

Reply:
I ended up adding a test for `get_connection_types` here. I decided not to add one for `connection_type_secret_schema` here because it takes connection specific information. It gets tested in other places and I thought this could be harder to maintain making sure all tests are using the same info. I could create a fixture for this, but en the end that would give us multiple tests testing the same thing so thought it wasn't worth it.